docs/csharp/language-reference/keywords/ref.md

```diff
@@ -105,6 +108,10 @@ ref VeryLargeStruct reflocal = ref veryLargeStruct;
 
 Note that in both examples the `ref` keyword must be used in both places, or the compiler generates error CS8172, "Cannot initialize a by-reference variable with a value."
 
+## ref readonly local
```

Comment:
To be consistent with "Ref locals" naming: "ref readonly local" -> "Ref readonly locals"